### DISCUSSION TITLE
Changing the channel models in Neural Receiver for OFDM SIMO Systems to UMi, RMa and UMa

### DISCUSSION THREAD

**Discussion by yhuwhai** [NVlabs/sionna#252] [Q&A] [answered]
Hi everyone,

Please can I get help with changing the channel model in this example ( https://nvlabs.github.io/sionna/examples/Neural_Receiver.html ) to any of RMa, UMi, and UMa.

I tried adapting the example in (https://nvlabs.github.io/sionna/examples/Realistic_Multiuser_MIMO_Simulations.html#System-Setup) to the one in Neural Receiver for OFDM SIMO.

I get this error :

```
Exception has occurred: OperatorNotAllowedInGraphError
Exception encountered when calling layer 'e2e_system' (type E2ESystem).

in user code:

    File "C:\Users\....\Umi_test.py", line 460, in call  *
        y,h = self._channel([x_rg, h_freq, no_]).
```

I can share my code.

Thanks in advance.

Uyoata

**Comment by faycalaa** [comment]
Hi,

Could you share a minimal code snippet that reproduces the error?

**Reply by yhuwhai** [reply]
Sorry for the very late reply. This had been solved.

**Comment by omsahoowork** [comment, marked as answer]
You can create a 3d channel example UMA like this and give it in the parameter of the  end to end model and make sure to keep the batch size same and you can use the model with these channel also.

```py
UMA = sionna.channel.tr38901.UMa(carrier_frequency=CARRIER_FREQUENCY,
                             o2i_model="low",
                             ut_array=UT_ARRAY,
                             bs_array=BS_ARRAY,
                             direction=DIRECTION,
                             enable_pathloss=False,
                             enable_shadow_fading=False)
```

Hope it helps :)
